addons/mail/static/tests/discuss_app/sidebar_tests.js

```diff
@@ -1034,7 +1034,7 @@ QUnit.test("chat - avatar: should have correct avatar", async (assert) => {
     await openDiscuss();
 
     assert.containsOnce($, ".o-mail-DiscussCategoryItem img");
-    assert.containsOnce($, `img[data-src='/web/image/res.partner/${partnerId}/avatar_128']`);
+    assert.containsOnce($, `img[data-src='/web/image/res.partner/${partnerId}/avatar_128?unique=undefined']`);
```

Comment:
Should use assignDefined so that we never end up in a situation where we set undefined as param